src/Chip/index.tsx

```diff
@@ -3,51 +3,74 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 // @ts-ignore
 import Icon from '../Icon';
+// @ts-ignore
+import IconButton from '../IconButton';
+
+export const ICON_AFTER_ALT_TEXT = 'Chip icon after';
```

Comment:
I don't think it's a good idea have default alt text here, it heavily depends on the context and I don't think there is a reasonable default. We should make alt text props required only when consumers user icons with `Chip`, otherwise default to `undefined` (if no icons are used, then we do not need alt text at all).

We can achieve this behaviour with [requiredWhen proptype](https://github.com/openedx/paragon/blob/master/src/utils/propTypes/utils.js#L34), e.g.
```js
Chip.propTypes = {
  ...
  iconBeforeAlt: requiredWhen(PropTypes.string, 'iconBefore')
}

Chip.defaultProps = {
  iconBeforeAlt: undefined,
};
```

Reply:
Corrected